doc/releases/release-notes-3.5.rst

```diff
@@ -35,6 +35,17 @@ Changes in this release
 * The Kconfig option CONFIG_GPIO_NCT38XX_INTERRUPT has been renamed to
   :kconfig:option:`CONFIG_GPIO_NCT38XX_ALERT`.
 
+* ``zephyr,memory-region-mpu`` was renamed ``zephyr,memory-attr`` and its type
+  moved from 'enum' to 'int'. To have a seamless conversion this is the
+  required change in the DT:
+
+    - "RAM"         -> <( DT_MEM_ARM(ATTR_MPU_RAM) )>
```

Comment:
Does this look OK in the html? I noticed that below this text is put in a code block, but that's not present here.

Reply:
does the release note rendered in html? In that case yes, the code block is missing.

Reply:
done